client/src/components/pages/home/components/reveal-on-scroll-cards/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTranslations } from 'next-intl';
+
+import { getStyles } from './styles';
+import { scrollItemsData } from './mock';
+import { ScrollCard } from './scroll-card';
+
+export const RevealOnScroll = () => {
+  const styles = getStyles();
+  const text = useTranslations('homePage.revealOnScroll');
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.titleWrapper}>
+        <span className={styles.titleText}>{text('title.firstPart')}</span>
+        <span className={styles.titleText}>{text('title.secondPart')}</span>
+        <span className={styles.titleText}>{text('title.thirdPart')}</span>
+      </div>
+
+      {scrollItemsData(text).map((item, index) => (
+        <ScrollCard key={item.id} text={item.text} number={item.id} isEven={index % 2 === 1} />
+      ))}
```

Comment:
Your id's is 1,2,3 you them into the key. Big chance to repeat this key. Change id or key